master/buildbot/data/forceschedulers.py

```diff
@@ -24,6 +24,7 @@
 def forceScheduler2Data(sched):
     ret = dict(all_fields=[],
                name=unicode(sched.name),
+               button_name=unicode(sched.buttonName),
```

Comment:
OK, I wasn't aware of it!
Looks good, but we need to fix the default value to other thing as "Force Build"